config/credentials.default.coffee

```diff
@@ -1,3 +1,5 @@
+secret = () -> Math.random().toString(36).slice(-16)
```

Comment:
I would vote for removing since deployers will need to manage two files for configuration of their deployment. But if majority of the votes say "lets have it" i would want it to be moved to main.default.coffee and have them passed with `options` into credential.dev.coffee

On the other hand we could make credentials.* files configurable easily since `./configure` passes all command line options to downstream